sdk/eventgrid/azure-eventgrid/swagger/postprocess_eventnames.py

```diff
@@ -22,12 +22,11 @@ def extract(definitions):
     return tups
 
 def generate_enum_content(tuples):
-    print("# these names below are for backward compat only - refrain from using them.")
-    for k, v in backward_compat.items():
-        print(k + " = '" + v + "'\n")
-    print("# backward compat names end here.")
     for tup in tup_list:
         print(tup[0] + " = '" + tup[1].replace('API', 'Api') + "'\n")
+    print("# these names below are for backward compat only")
+    for k, v in backward_compat.items():
+        print(k + " = '" + v + "'\n")
```

Comment:
These shouldn't be at the bottom of the enum right? :P

Reply:
updated it :P